app/backend/src/clj/teet/backup/backup_ion.clj

```diff
@@ -364,6 +382,7 @@
              prepare-asset-db-for-restore
              restore-tx-file
              migrate
+             log-restore-result
```

Comment:
what about exception case? I think failure should be logged as well.. so anyone polling for this log file knows that the thing failed

Reply:
- Exception in saving file can not be logged as something already wrong in saving to S3 bucket.
- The exception in restore thowns Exception. In this case the log-restore-result function will not be called at all. 
- Is there any other case then failure in restore saved in context, for instance, without throwing an exception?